app/src/modules/settings/routes/data-model/fields/components/field-select.vue

```diff
@@ -17,7 +17,23 @@
 				</div>
 			</template>
 		</v-input>
+		<v-input v-if="isDisabled" disabled class="field">
+			<template #prepend>
+				<v-icon v-tooltip="t('system_fields_locked')" name="lock" />
+			</template>
 
+			<template #input>
+				<div
+					v-tooltip="`${field.name} (${formatTitle(field.type)})${interfaceName ? ` - ${interfaceName}` : ''}`"
+					class="label"
+				>
+					<div class="label-inner">
+						<span class="name">{{ field.field }}</span>
+						<span v-if="interfaceName" class="interface">{{ interfaceName }}</span>
+					</div>
+				</div>
+			</template>
+		</v-input>
```

Comment:
I think this might mess with the `v-else` on line 37. You should be able to just add the ne condition to line 3:

```html
<v-input v-if="disabled || isDisabled" disabled class="field">
```

Reply:
Updated